### PR TITLE
Bootstrap 4.6.0

### PR DESCRIPTION
**Pull request by agriffard**
https://github.com/twbs/bootstrap/releases/tag/v4.6.0
https://www.npmjs.com/package/bootstrap

### CODE REVIEW

**Comment by Skrypt** [comment]
Have you tried it with the frontend themes? Are they not individually "compiled" with their own assets and BS version?

**Comment by agriffard** [comment]
I haven't changed the Agency, Blog and Coming soon themes yet.

File: src/OrchardCore.Themes/TheAgencyTheme/package.json [hunk 1]
```diff
@@ -2,6 +2,6 @@
   "name": "orchardcore.theagencytheme",
   "version": "1.0.0",
   "dependencies": {
-    "bootstrap": "4.5.3"
+    "bootstrap": "4.6.0"
```
Comment (Skrypt) [inline src/OrchardCore.Themes/TheAgencyTheme/package.json:5]: That's one change to TheAgency ... ?

**Comment by Skrypt** [comment]
What I meant is if you look there : 
https://github.com/startbootstrap/startbootstrap-agency
He's at bootstrap 4.5.3 ...

**Comment by agriffard** [comment]
Unexpected merge: I followed the instructions to solve the conflicts with the command line and the PR branch has been merged to dev instead of the opposite.

It should be ok but tell me if it has to be reverted.

The StartBootstrap templates are still using 4.5.3.

**Pull request merged**